modules/ting/ting.client.inc

```diff
@@ -142,55 +143,53 @@ function ting_get_objects($ids) {
     }
   }
 
-  // Not all object are searchable, such as relation etc. So to get over this we
-  // split the request into to groups "own id's" and "others". Where the first
-  // is ensured to be searchable.
-  $agency = variable_get('ting_agency', FALSE);
-  $query = array();
   foreach ($objects as $id => $object) {
     if ($object === FALSE) {
-      // So if the agency match lets search theme as that's faster then fetching
-      // them one by one.
-      if (preg_match('/^(890790-basis|' . $agency . '-katalog|' . $agency . ')/', $id)) {
-        $query[] = 'rec.id=' . $id;
-      }
-      else {
-        // Get objects as it was not local.
-        $objects[$id] = ting_get_object($id);
-      }
+      // Collect ids to getObjectRequest
+      $get_ids[] = $id;
     }
   }
-  // Open search is limited to 50 results per call, so iterate until all results
-  // have been fetched. It has a limit on the size of the query (>187 rec.id=
-  // ORed together seems to break it).
```

Comment:
default limit is 200 manifestations. It can be set in opensearch's configuration